pkg/tinydtls/Makefile

```diff
@@ -1,12 +1,19 @@
 PKG_NAME=tinydtls
 PKG_URL=https://github.com/rfuentess/TinyDTLS.git
-# PKG_VERSION=RIOT-OS
-PKG_VERSION=d06ad84d3e8bd3e86f7557faee34b68d5f8c0129
+#Last tested commit:
+PKG_VERSION=5b5ba4b3c7a79dda168103be4d119ca489f89c3c
+# (Official) Branch with tested(?) updates before merging to remote master
+#PKG_VERSION=RIOT-Tested-Upgrades
```

Comment:
A question, I was leaving that line for having an easier time for testing the future upgrades to tinyDTLS. Also, for those users that requires to have access to recent patches that would still be in the process of being tested. Is not recommended having it?